Header header
uint8 controller_type
uint8 setpoint_trajectory
float32 setpoint_frequency
float32 setpoint_mean
float32 setpoint_amplitude
bool shutdown
